### DISCUSSION TITLE
Midinous to MiniDexed -- HOW?

### DISCUSSION THREAD

**Discussion by akivaatwood** [probonopd/MiniDexed#525] [Q&A] [answered]
On OSx, I have midinous running. It send midi signals to IAC drivers. The minidexed doesn't show up as a midi device...

Anyone have any suggestions on how to get them talking?

Thanks

**Comment by probonopd** [comment, marked as answer]
You need a device that presents itself to your Mac like a MIDI controller (keyboard), and that sends MIDI data to the MiniDexed device. 

https://github.com/probonopd/MiniDexed/wiki/Hardware#usb-midi-device

Have a look at https://github.com/rsta2/pico/tree/main/midi_adapter, it might serve your needs.

Related discussion:
* https://github.com/probonopd/MiniDexed/discussions/452

**Reply by akivaatwood** [reply]
Thanks -- that was what I suspected. Luckilly I have a second Pi4....

Though I'll see if there's an arduino version

Thanks

**Reply by akivaatwood** [reply]
Just a thought -- I have an arcturia 37 which has midi in and out -- shouldn't that work as an adapter between the PC and the minidexed?

**Reply by probonopd** [reply]
Just keep in mind that while you can connect USB MIDI controllers (keyboards) to MiniDexed, MiniDexed cannot act like an instrument over USB (this works only over serial).

**Reply by diyelectromusic** [reply]
Here is one solution using a Seeed XIAO on a custom PCB: https://github.com/probonopd/MiniDexed/discussions/480
And here are a number of generic solutions using an Arduino: https://diyelectromusic.wordpress.com/2022/03/22/arduino-and-usb-midi/

Kevin
